source/unit/Attack.gd

```diff
@@ -8,4 +8,34 @@ export var damage := 0
 export var strikes := 0
 export var icon : StreamTexture = null
 
-onready var specials := get_children()
\ No newline at end of file
+onready var specials := get_children()
+
+func execute_before_turn(user, target):
+	if not specials.empty():
+		for child in specials:
+			if child.has_method("execute_before_turn_to_ennemy"):
```

Comment:
you changed the function names but not the strings here. :)

Reply:
@Byteron ok, I solved this :)